subprojects/zap-clientapi/src/main/java/org/zaproxy/clientapi/core/Alert.java

```diff
@@ -31,6 +31,7 @@
 	public enum Reliability {Suspicious, Warning};
 	public enum Confidence {Low, Medium, High, Confirmed};

```

Comment:
If these are going to be returned/converted shouldn't they align with or simply use the core of enums in Alert?